src/core/domain/DTOs/patch_info_dto.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, constr
+from pydantic_factories import ModelFactory
+
+
+class status_enum(str, Enum):
+    active = "active"
+    resolved = "resolved"
+    muted = "muted"
+
+
+class level_enum(str, Enum):
+    critical = "critical"
+    error = "error"
+    warning = "warning"
+    info = "info"
+    debug = "debug"
```

Comment:
In our project, we have accorded that key enums are always in Uppercase

In this case:

```python
class LevelEnum(str, Enum):
    CRITICAL = "critical"
```